src/migrations/20191118191222-create-room.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('rooms', {
```

Comment:
This should have a column to specify whether a room is booked or not, I'd suggest a boolean value for that. I see that you used an extra table to provide info about rooms that are already booked. My problem with this approach is, think about this: How would you fetch rooms that are not yet booked?

Reply:
You are right, that would bring more complexities.